posix.mak

```diff
@@ -210,10 +210,10 @@ $(DOC_OUTPUT_DIR)/sitemap.html : $(ALL_FILES_BUT_SITEMAP) $(DMD)
 	cp -f sitemap-template.dd sitemap.dd
 	(true $(foreach F, $(TARGETS), \
 	  && echo \
-        "$F\t`sed -n 's/<title>\(.*\) - D Programming Language.*<\/title>/\1/'p $(DOC_OUTPUT_DIR)/$F`")) \
+	    "$F	`sed -n 's/<title>\(.*\) - D Programming Language.*<\/title>/\1/'p $(DOC_OUTPUT_DIR)/$F`")) \
```

Comment:
> It's a tab followed by 4 spaces, shouldn't it be 2 tabs instead?

No, because the tab is part of the Makefile syntax, and the spaces are for indentation - this rule is indented with an indentation size of 2. Regardless, I didn't touch that part of the line :P